charts/gitops-server/README.md

```diff
@@ -0,0 +1,52 @@
+# Weave Gitops Helm Chart
+
+This is the [Weave Gitops](https://github.com/weaveworks/weave-gitops) [Helm](https://helm.sh) chart.
+
+It installs the weave-gitops-server component as a 1-replica deployment.
+
+Optionally it will also install:
+* Service Account
+* ClusterRoleBinding (to the service account) and ClusterRole with the
+  permissions required to run Gitops.
+* Service, this is optional as you may want to limit access to the UI to via
+  port-forwarding
+* Ingress
+* Test User -- A test user with hard-coded username & password with minimal
+  permissions
+
+This chart assumes kubernetes > 1.17
+
+## Security
+
+The role that this chart creates includes 2 main 'blocks' of permissions; that
+should be treated separately and carefully:
+
+* `impersonate` This is how the gitops-server gathers data to display in the UI,
+  it impersonates the user, determined by OIDC/plain auth. This means that
+  a user's permissions in the UI will reflect their permissions in the cluster
+* `get`, `list`, `watch` on `helmrepositories` and `secrets`. These permissions
+  are required by the profiles system.
+
+### Impersonate
+
+When deploying gitops-server it is recommended to limit the types of resource
+and specific resources that the service account can impersonate. e.g.
+```yaml
+rbac:
+  create: true
```

Comment:
It is not like users have a choice here, they must have impersonation setup otherwise nothing works. So ideally we could have some default values as `impersonaionResouces` and `impersonationResourceNames` but allow configuration of those here.

Reply:
I've improved how the defaults for `impersonationResources` is set in ca342853 

Much as I'd love to provide defaults for `impersonationResourceNames` I can't think of a good way to do so. The best solution I've come up with would be to create some default groups (e.g. `viewer` and `editor`) that are then the only things to be impersonated. The problem with this is that it requires the user's k8s authentication system to be configured to support them. I suspect this is best solved via documenting common processes for it but that feels very out of the scope of this PR (I'm also not 100% sure this is the best way to do it, I'm not sure who might be the best person to cover this sort of security work).

Reply:
definitely, we can't set anything there since there is no way for us to know what groups the companies use. I'm good with that, the important bit that's allowing configuration is there, it should be enough.